db/migrate/20250216134558_remove_hash_secret_from_auth_tokens.rb

```diff
@@ -0,0 +1,17 @@
+class RemoveHashSecretFromAuthTokens < ActiveRecord::Migration[8.0]
+  def up
+    remove_column :auth_tokens, :hashed_secret, :string, null: false
+  end
+
+  def down
+    add_column :auth_tokens, :hashed_secret, :string, null: true
+
+    AuthToken.find_each do |at|
+      at.secret = SecureRandom.urlsafe_base64(48)
+      at.hashed_secret = BCrypt::Password.create(secret, cost: Rails.configuration.token_hash_rounds)
+      at.save!
+    end
+
+    change_column_null :auth_tokens, :hashed_secret, false
+  end
```

Comment:
Not sure about this `down` migration. The hashed_secret will not work. I think it would be more honest to just delete all the existing auth tokens.

Reply:
(Also, the configuration for `token_hash_rounds` should probably be removed, since it isn't used in the actual application anymore.)

Reply:
I do think we should remove the configuration, and just hardcode the cost here

Reply:
Sure.